src/chester.js

```diff
@@ -64,6 +64,7 @@ module.exports.Chester = (secret: string | Buffer, {
       throw new TypeError();
     }
 
+    // what's the rationale for storing these in fixed widths at the end of the message?
```

Comment:
Makes it really easy to extract again from the decrypted message. Otherwise you need to store information about how long a part is.

Reply:
It's a size thing no? Otherwise it could just be an envelope wrapper on the message body if that wasn't a concern...?

Reply:
Ah, I see what you're saying. However we need a buffer that can be encrypted. So if we use an envelope wrapper we still need to convert that into a buffer somehow. I'm not sure how that would best be done. But yes, if size wasn't a concern and it was clear how one would convert an envelope (e.g an object) into a buffer and back this could be done.